### PR TITLE
Makes space 250 times less potent

### PR DESCRIPTION
**Pull request by Razharas**
Fix https://github.com/tgstation/-tg-station/issues/17814

### CODE REVIEW

**Comment by optimumtact** [comment]
closed, we're in a feature freeze

**Comment by optimumtact** [comment]
Just kidding babe

**Comment by Razharas** [comment]
You are breaking travis oranges

**Comment by hornygranny** [comment]
i like this bug

File: code/modules/atmospherics/gasmixtures/space_mixture.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 	return 0 //we're immutable.
 
 /datum/gas_mixture/space/heat_capacity()
-	. = 700000
```
Comment (Firecage) [inline code/modules/atmospherics/gasmixtures/space_mixture.dm]: PR title is faulty. I did the calculations and this only makes it 100x less potent.

**Pull request merged**